### PR TITLE
Backup restore op deployment

### PR DESCRIPTION
**Pull request by mantis-toboggan-md**
#1019 
Also, updated the display name to 'Backup & Restore Operator' for #1186 
Screen Shot 2020-09-14 at 11 23 47 AM
Screen Shot 2020-09-14 at 11 23 58 AM
Screen Shot 2020-09-14 at 11 24 11 AM

### CODE REVIEW

**Comment by mrajashree** [comment]
@mantis-toboggan-md we no longer need the "Rancher ResourceSet Namespace" field/dropdown. We decided to remove it after I had created the UI ticket, I will update the issue now

**Comment by mantis-toboggan-md** [comment]
Updated to remove rancher resourceset namespace:
Screen Shot 2020-09-14 at 11 57 06 AM

**Comment by mrajashree** [comment]
@mantis-toboggan-md Should we add a helper text/note below/above the header "Default Storage Location", that says
"Configure a storage location where all backups can be saved by default. You will have the option to override this per backup to use a different s3-compatible object store"

I think we discussed that we don't need to show operator image and tag on the form? cc @deniseschannon

**Comment by mantis-toboggan-md** [comment]
Updated to remove operator image and tag,  and add tip about overriding default storage:
Screen Shot 2020-09-15 at 8 58 19 AM

**Comment by mantis-toboggan-md** [comment]
Updated tip text: 
Screen Shot 2020-09-15 at 11 38 51 AM

File: chart/backup-restore-operator/index.vue [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+    storageSource(neu) {
+      switch (neu) {
+      case 'defaultSC':
+        this.value.persistence.enabled = true;
+        this.value.s3.enabled = false;
+        this.value.persistence.storageClass = this.defaultStorageClass.id;
+        break;
+      case 'pickSC':
+        this.value.persistence.enabled = true;
+        delete this.value.persistence.volumeName;
```
Comment (mrajashree) [inline chart/backup-restore-operator/index.vue:113]: this should set storageClass in values.yaml to whichever storageClass user selects

File: chart/backup-restore-operator/index.vue [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+        this.value.s3.enabled = false;
+        break;
+      case 'pickPV':
+        this.value.persistence.enabled = true;
+        delete this.value.persistence.storageClass;
```
Comment (mrajashree) [inline chart/backup-restore-operator/index.vue:118]: in this case, the storageClass should be set to `"-"` in values.yaml

**Pull request merged**